EZEE-3224: Segmentation API (#1172)
ezsystems/developer-documentation: docs/guide/admin_panel.md

Object states can be used in conjunction with permissions, in particular with the [State Limitation](limitation_reference.md#state-limitation).
Their specific use cases depend on your needs and the setup of your permission system.

## Segments

> Segments are used to display specific content to specific Users.

Reviewer: We might consider going for the active voice here:
"You use segments to display ..."

> Segments are used to display specific content to specific Users.

Author: ```suggestion
You can use Segments to display specific content to specific Users.
```
